apps/dialtone-documentation/docs/about/whats-new/posts/2024-3-20.md

```diff
@@ -0,0 +1,200 @@
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">body-compact</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">body-md-compact</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">body-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">body-sm</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">body-compact-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">body-sm-compact</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-base</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-md</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-compact</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-md-compact</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-sm</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-plain</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-md-plain</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-compact-plain</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-md-plain-compact</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-compact-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-sm-compact</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-plain-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-sm-plain</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">label-compact-plain-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">label-sm-plain-compact</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">helper-base</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">helper-md</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">helper-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">helper-sm</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">code-base</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">code-md</code></td>
+      </tr>
+      <tr>
+        <td class="d-bgc-black-100"><code class="d-code--sm">code-small</code></td>
+        <td><code class="d-code--sm d-fc-purple-400">code-sm</code></td>
+      </tr>
+    </tbody>
```

Comment:
Both columns: pre `d-` to each CSS Utility.

New column: note the naming convention's use of `--` (double-dash) after the category. e.g. `headline-sm-soft` → `d-headline--sm-soft`.